modules/host/host.go

```diff
@@ -2,13 +2,12 @@ package host
 
 import (
 	"errors"
+	"log"
 	"net"
-	"os"
 	"sync"
 
 	"github.com/NebulousLabs/Sia/crypto"
 	"github.com/NebulousLabs/Sia/modules"
-	"github.com/NebulousLabs/Sia/modules/consensus"
 	safesync "github.com/NebulousLabs/Sia/sync"
```

Comment:
probably okay to move the whole package to just "sync", safesync probably not needed anymore